content/200-concepts/100-components/01-prisma-schema/06-relations/100-one-to-one-relations.mdx

```diff
@@ -43,12 +49,55 @@ model Profile {
 </tab>
 </TabbedContent>
 
-The `userId` relation scalar is a direct representation of the foreign key in the underlying database.This 1-1-relation expresses the following:
+The `userId` relation scalar is a direct representation of the foreign key in the underlying database. This one-to-one relation expresses the following:
 
 - "a user can have zero or one profiles" (because the `profile` field is [optional](/concepts/components/prisma-schema/data-model#type-modifiers) on `User`)
 - "a profile must always be connected to one user"
 
-</TopBlock>
+In this example, the `user` field of the `Profile` model references the `id` field of the `User` model. It is also possible to reference another field, as long as that field is marked with the `@unique` attribute. This is necessary to guarantee that there is only a single `User` connected to each `Profile`. In the following example, the `user` field references an `email` field in the `User` model, which is marked with the `@unique` attribute:
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma
+model User {
+  id      Int      @id @default(autoincrement())
+  email   String   @unique // <-- add unique attribute
+  profile Profile?
+}
+
+model Profile {
+  id     Int  @id @default(autoincrement())
+  user   User @relation(fields: [userId], references: [email])
+  userId Int  @unique // relation scalar field (used in the `@relation` attribute above)
+}
+```
+
+</tab>
+<tab>
+
+```prisma
+model User {
+  id      String   @id @default(auto()) @map("_id") @db.ObjectId
+  email   String   @unique // <-- add unique attribute
+  profile Profile?
+}
+
+model Profile {
+  id     String @id @default(auto()) @map("_id") @db.ObjectId
+  user   User   @relation(fields: [userId], references: [email])
+  userId String @unique @db.ObjectId // relation scalar field (used in the `@relation` attribute above)
+}
+```
+
+</tab>
+</TabbedContent>
+
+<Admonition type="warning">
+
+In MySQL, it is possible to create a foreign key with only an index on the referenced side, and not a unique constraint. In Prisma versions 4.0.0 and later, if you introspect a relation like this it will trigger a validation error. To fix this, you will need to add a `@unique` constraint to the referenced field.
```

Comment:
```suggestion
In MySQL, you can create a foreign key with only an index on the referenced side, and not a unique constraint. In Prisma versions 4.0.0 and later, if you introspect a relation like this it will trigger a validation error. To fix this, you will need to add a `@unique` constraint to the referenced field.
```